Yona/Yona/BaseViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BaseViewController.swift
+//  Yona
+//
+//  Created by Chandan on 19/05/16.
+//  Copyright © 2016 Yona. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(BaseViewController.foreground), name: UIApplicationDidBecomeActiveNotification, object: nil)
+    }
```

Comment:
Why do we need such event in a base view controller?
Also, when do we remove the view controller from the notification center?